packages/reports/reports.hjson

```diff
@@ -0,0 +1,212 @@
+{
+  "reports": [
+    {
+    "id": "8",
+    "config": {
+      "fetch": [
+        {
+          "fetchAnalytics": {
+            "dataElementCodes": [
+              "BCD46",
+              "BCD47",
+              "BCD48",
+              "BCD48a",
+              "BCD48b",
+              "BCD49",
+              "BCD50",
+              "BCD51",
+              "BCD52",
+              "BCD53",
+              "BCD54",
+              "BCD55"
+            ],
+            "aggregationType": "RAW",
+            "dataServices": [
+              {
+                "isDataRegional": true
+              }
+            ]
+          }
+        }
+      ],
+      "transform": [
+        {
+          "transform": "select",
+          "$.dataElement": "$.value",
+          "...": ["period", "organisationUnit"]
+        },
+        {
+          "transform": "sort",
+          "by": "$.period",
+        },
+        {
+          "transform": "aggregate",
+          "organisationUnit": "group", 
+          "...": "last"
+        },
+        {
+          "transform": "aggregate",
+          "organisationUnit": "drop",
+          "period": "drop",
+          "...": "sum"
+        },
+        {
+          "transform": "select",
+          "'Doctors'": "$.BCD46",
+          "'Midwives'": "$.BCD47",
+          "'Nurses'": "add([$.BCD48, $.BCD48a, $.BCD48b])",
+          "'Aides'": "$.BCD49",
+          "'Others'": "add([$.BCD50, $.BCD51, $.BCD52, $.BCD53, $.BCD54, $.BCD55])"
+        }
+      ],
+      "output": [
+          {
+            "Doctors": 2,
+            "Midwives": 1,
+            "Nurses": 12,
+            "Aides": 7
+          }
+      ]
+    }
+  },
+  {
+    "id": "TO_RH_Descriptive_FP01_02",
+    "config": {
+      "fetch": [
+        {
+          "fetchAnalytics": {
+            "dataElementCodes": [
+              "Family_Planning_Acceptors_Other",
+              "Family_Planning_Acceptors_Condom_Female",
+              "Family_Planning_Acceptors_Natural_Method",
+              "Family_Planning_Acceptors_Depo_Provera",
+              "Family_Planning_Acceptors_Jadelle",
+              "Family_Planning_Acceptors_Pill_Single",
+              "Family_Planning_Acceptors_Sterilization_Male",
+              "Family_Planning_Acceptors_Sterilization_Female",
+              "Family_Planning_Acceptors_Pill_Combined",
+              "Family_Planning_Acceptors_Condom_Male",
+              "Family_Planning_Acceptors_IUD",
+              "POP21AND23",
+              "POP15",
+              "POP17AND19",
+              "POP13",
+              "POP11"
+            ],
+            "aggregationType": "RAW",
+            "dataServices": [
+              {
+                "isDataRegional": false
+              }
+            ]
+          }
+        }
+      ],
+      "transform": [
+        {
+            "transform": "select",
+            "$.dataElement": "$.value",
+            "'period'": "$.period"
```

Comment:
I don't understand this line...

Reply:
Yup, you're right, but using `'` is just what I've had to do currently to get mathjs to parse the expression currently. Ideally we'd check the string and see if it contains a `$` (or something else that requires parsing) and determine whether to parse it at all. That way we wouldn't need to wrap literals in `'` in these cases.